src/symbols/analyzer.rs

```diff
@@ -1177,19 +1189,34 @@ impl ModuleBuilder {
       symbol_id
     } else {
       let symbol_id = self.get_next_symbol_id();
-      let symbol = SymbolMut::new(Symbol::new(self.module_id, symbol_id));
+      let symbol = SymbolMut::new(Symbol::new(
+        self.module_id,
+        symbol_id,
+        Some(module_symbol.symbol_id()),
+      ));
       symbol.add_decl(symbol_decl);
       self.symbols.insert(symbol_id, symbol);
       module_symbol.add_export("default".to_string(), symbol_id);
+      module_symbol.add_child_id(symbol_id);
```

Comment:
No need to review this code with too much scrutiny. I wrote some tests in test_builder.rs to help verify the output.